fbw-a32nx/src/systems/instruments/src/SD/Pages/Cond/Cond.tsx

```diff
@@ -13,20 +13,30 @@ export const CondPage = () => {
     // Display trim valve position for each zone
     const gaugeOffset = -43; // Gauges range is from -43 degree to +43 degree
 
-    const [cockpitTrimAirValve] = useSimVar('L:A32NX_COND_CKPT_TRIM_AIR_VALVE_POSITION', 'number', 1000);
-    const [cockpitTrimTemp] = useSimVar('L:A32NX_COND_CKPT_DUCT_TEMP', 'celsius', 1000);
+    const Acsc1DiscreteWord1 = useArinc429Var('L:A32NX_COND_ACSC_1_DISCRETE_WORD_1');
+    const Acsc2DiscreteWord1 = useArinc429Var('L:A32NX_COND_ACSC_2_DISCRETE_WORD_1');
+    const AcscDiscreteWord1 = !Acsc1DiscreteWord1.isFailureWarning() ? Acsc1DiscreteWord1 : Acsc2DiscreteWord1;
```

Comment:
**style:** variable identifiers that are not React/FSComponent components should start with a lowercase letter

Reply:
Fixed, thanks!